hydra_redis/redisgraph_demo1.py

```diff
@@ -0,0 +1,348 @@
+import redis
+from redisgraph import Graph, Node, Edge
+import urllib.request as ur
+import json
+import re
+from re import compile as regex
+from httplib2 import Http
+from hydrus.hydraspec import doc_maker
+
+
+redis_con = redis.Redis(host='localhost', port=6379)
+redis_graph = Graph("apidoc", redis_con)
+
+
+class _MemCache(dict):
+
+    def __nonzero__(self):
+        # even if empty, a _MemCache is True
+        return True
+
+    def set(self, key, value):
+        self[key] = value
+
+    def delete(self, key):
+        if key in self:
+            del self[key]
+
+
+DEFAULT_HTTP_CLIENT = Http(_MemCache())
+http = DEFAULT_HTTP_CLIENT
+APIDOC_RE = regex(
+    r'^<([^>]*)>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"$')
+
+
+def final_file(url):
+    response = ur.urlopen(url)
+    new_file = json.loads(response.read().decode('utf-8'))
+    return new_file
+
+
+def get_method(new_obj):
+    obj_method = []
+    for obj in new_obj:
+        obj_method.append(obj["method"].lower())
+    return obj_method
+
+
+def get_title(new_obj):
+    new_obj = new_obj[0]
+#    print (new_obj)
+    obj_title = new_obj["description"]
+    print(obj_title)
+    return str(obj_title[4:])
+
+
+def get_endpoint_id(string):
+    return (string.replace("vocab:EntryPoint", ""))
```

Comment:
**As I've pointed out multiple times please use Doc Strings for every function you write.**